### ISSUE TITLE
Power-method annotation omits the theorem’s invariant initialization, population loss, and RMS normalization

### ISSUE THREAD

**Issue by microprediction** [microprediction/latentreasoning#49]
## Reviewed source

- Commit: [`ef9bb829b8866cf29902f6093fcbb4b71126d0dc`](https://github.com/microprediction/latentreasoning/commit/ef9bb829b8866cf29902f6093fcbb4b71126d0dc)
- [`bibliography.html`, Wu–Zhang–Cao entry](https://github.com/microprediction/latentreasoning/blob/ef9bb829b8866cf29902f6093fcbb4b71126d0dc/bibliography.html#L469-L472)

The annotation says:

> A looped linear transformer with LayerNorm, trained by gradient descent, provably converges to power-method iteration on a synthetic testbed.

“Synthetic testbed” is useful scope, but it does not communicate the assumptions doing the work, and “with LayerNorm” is not the normalization analyzed.

## What Theorem 4.1 actually proves

The theorem concerns a deliberately structured principal-component predictor:

- **No softmax:** §3.2 explicitly removes the softmax nonlinearity and uses linear attention.
- **Not standard LayerNorm:** each column is merely divided by its Euclidean norm. The paper explicitly calls this structurally equivalent to **RMSNorm**, not mean-centered LayerNorm.
- **Special input representation:** the data matrix and a random unit query occupy disjoint blocks of a `2d × (n+1)` input.
- **Special initialization:** `W(0)=0`; only the `V21` block is nonzero and is initialized to the identity. This puts gradient descent on an invariant two-scalar manifold where the active blocks remain `w_t I` and `v_t I`.
- **Population gradient descent:** the objective is an expectation over the data and random query, not finite-sample SGD.
- **Strong data symmetry:** all input columns have unit norm; the data distribution is invariant under every orthogonal rotation; and the inverse smallest eigenvalue is integrable.
- **Learning-rate/dimension conditions:** `d >= 3` and `0 < eta <= d/(2Ln)`.

Under those assumptions the predictor at training step `t` has the exact form

```
normalize((I + rho_t X X^T)^L a),  where rho_t = w_t v_t.
```

As `rho_t -> infinity`, its **function** approaches the normalized `L`-step power-method output `normalize((X X^T)^L a)`.

The parameters themselves do not converge: Theorem 4.1 gives `w_t, v_t = Theta(t^(1/4))`, so they diverge. The population loss converges to a **strictly positive** limiting loss, not zero.

## Why this matters

The current sentence can be read as an optimization result for ordinary looped transformers with LayerNorm. The proof instead relies on an uncentered normalizer, softmax-free attention, a hand-aligned representation and initialization, population gradients, and rotational symmetry. Those assumptions are precisely what preserve the two-scalar invariant subspace from which the power iteration follows.

This remains an interesting algorithmic-implicit-bias result, but it is not evidence that standard looped LLM training generally selects the power method.

## Suggested correction

For example:

> In a stylized PCA problem, full-batch population gradient descent from a prescribed block initialization makes a softmax-free, weight-tied linear-attention model with columnwise RMS normalization approach the finite-`L` power-method predictor. The result assumes rotation-invariant unit-norm data and an integrability condition; the weights diverge while the normalized predictor converges to a positive-loss limit.

Primary source: [Wu, Zhang, and Cao, arXiv:2606.00605v1](https://arxiv.org/html/2606.00605), especially Assumption 3.2, §3.2, and Theorem 4.1.

**Comment by microprediction** [comment]
Correct, fixed in 99fd23f.

Verified against arXiv:2606.00605, and nearly every item maps to a verbatim sentence. The softmax is removed. The normalisation is not LayerNorm: the paper says it adopts "a formulation structurally equivalent to the RMSNorm", dropping mean-centering, which is the detail the annotation got flatly wrong. Initialisation is W=0 with only V21 set to the identity. The loss is a population expectation over data and query, not SGD. Assumption 3.2 requires unit-norm columns, rotation invariance under every orthogonal matrix, and integrable inverse smallest eigenvalue, with d at least 3.

And the ending matters: the training loss "converges to a strictly positive value", so this is convergence to power-method behaviour rather than to the task.

The annotation now names the linear architecture, the RMSNorm-equivalent normalisation, the population objective, the initialisation, the data assumptions and the positive limit.
